robot/rospackages/src/mcu_control/scripts/MockPdsNode.py

```diff
@@ -0,0 +1,283 @@
+#! /usr/bin/env python3
+
+import time
+import rospy
+import sys
+import random
+from std_msgs.msg import Float32
+from geometry_msgs.msg import Point
+from mcu_control.msg import ThermistorTemps, Voltage, Currents
+from decimal import *
+
+
+#Log all data during publishing
+#Add to publish_mock_data to log published data in CLI
+#Remove from publish_mock_data if not needed
+#Used for testing purposes
+def rospyPublishedValueLogger(prevParameterStates, prevParameterValues):
+    #Log new values when posted
```

Comment:
FYI this is not how you do pythondoc comments 

https://www.python.org/dev/peps/pep-0257/

Reply:
there's other instances of comments not using proper docstrings, I wont comment on them but this implies you should change them all

Reply:
I applied a change, not 100% sure if it is right though, but let me know once you see the update.

Reply:
Yeah, except that the comments are actually UNDER the function in python lol. You'll eventually get used to it

Reply:
Whoops yeah, i wasn't too sure when i read the docs. I'll change this. Bruh moment